### PR TITLE
Implementing manageColateral

### PR DESCRIPTION
**Pull request by tinystarinagalaxy**

### CODE REVIEW

**Review by DimaCantemir — APPROVED** [review]
Other than listing, LGTM

File: bxsolana/provider/http.py [hunk 1]
```diff
@@ -353,47 +361,28 @@ async def get_user(
...
         self,
         *,
         owner_address: str = "",
+        account_address: str = "",
         amount: float = 0,
         project: proto.Project = proto.Project.P_DRIFT,
-        contract: PerpContract = PerpContract.ALL,
-    ) -> proto.PostWithdrawCollateralResponse:
-        request = proto.PostWithdrawCollateralRequest()
+        type: PerpCollateralType = PerpCollateralType.PCT_DEPOSIT,
```
Comment (kdonthi) [inline bxsolana/provider/http.py:371]: I don't think this should have a default

File: bxsolana/provider/http.py [hunk 2]
```diff
@@ -353,47 +361,28 @@ async def get_user(
...
         request.project = project
         request.owner_address = owner_address
+        request.account_address = account_address
         request.amount = amount
-        request.contract = contract
+        request.type = type
+        request.token = token
         async with self._session.post(
-            f"{self._endpoint}/trade/perp/collateral/withdraw",
+            f"{self._endpoint}/trade/perp/managecollateral",
```
Comment (kdonthi) [inline bxsolana/provider/http.py:382]: I know you can't fix it here, but probably perp/manage-collateral or perp/collateral/update

Comment (tinystarinagalaxy) [reply bxsolana/provider/http.py:382]: thats what Daniel suggested

**Review by kdonthi — APPROVED** [review]

**Pull request merged**